cloud-minecraft/cloud-velocity/src/main/java/cloud/commandframework/velocity/CloudInjectionModule.java

```diff
@@ -67,6 +68,25 @@ public CloudInjectionModule(
         this.backwardsCommandSenderMapper = backwardsCommandSenderMapper;
     }
 
+    /**
+     * Create a new child injection module using Velocity's {@link CommandSource} as the sender type.
+     *
+     * @param commandExecutionCoordinator Command execution coordinator
+     * @return new injection module
+     * @since 1.5.0
+     */
+    public static @NonNull CloudInjectionModule<@NonNull CommandSource> createNative(
```

Comment:
Does this want to be added directly to the `VelocityCommandManager` as well?

Reply:
We encourage using the Injector anyways, in my opinion it's not necessary.